src/scripts/main.js

```diff
@@ -2,7 +2,22 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard tbody');
 
-// write your code here
+function addingRows(list, data) {
+  return list.forEach(person => {
+    return data.insertAdjacentHTML('beforeend',
```

Comment:
Так як це forEach можеш не повертати значення
```suggestion
    data.insertAdjacentHTML('beforeend',
```